translations/en/LC_MESSAGES/messages.po

```diff
@@ -145,6 +145,10 @@ msgstr ""
 "finish with a quotation mark. You forgot that for the text "
 "{unquotedtext}."
 
+msgid "Misspelled At Command"
```

Comment:
Hi @akseron!

It is not good practice to add to the po files directly, that will create problems. Instead, you have to add the new exception here: 

https://github.com/hedyorg/hedy/blob/5ba17c5c738ecb015e68b2e05dd77c5429dd6d0c/content/error-messages.txt#L10

And then run the extract script (see wiki) and the strings will be entered into the pot and po files automatically.

Reply:
See here: https://github.com/hedyorg/hedy/wiki/Hedy-Development-Process#adding-new-translation-keys